fdbclient/vexillographer/fdb.options

```diff
@@ -109,6 +109,9 @@ description is not currently required but encouraged.
             description="Searches the specified path for dynamic libraries and adds them to the list of client libraries for use by the multi-version client API. Must be set before setting up the network." />
     <Option name="disable_local_client" code="64"
             description="Prevents connections through the local client, allowing only connections through externally loaded client libraries. Intended primarily for testing." />
+    <Option name="client_threads_per_version" code="65"
```

Comment:
I will get back on this, but IIRC there was some weirdness around running vexillographer manually for bindings. I think it was just for Go, as the way Go imports works is that it clones the entire repo, instead of fetching a release tarball (like Ruby gems, Java jars etc ..).

Reply:
I think the thing that's weird is that the Go bindings are the only ones that have (or had?) the generated source checked in.

Reply:
Yes, it's because Go module management points to the actual repository/branch, and checks out the repo in go workspace instead of fetching a packaged bundle. So it has no (at-least idiomatic way) to build those once its checked out by module system. For Python, Java, Ruby,  its pre-built package e.g.  *.gem, *.egg, *.jar.